.github/workflows/build-test-deploy.yml

```diff
@@ -1101,26 +1101,6 @@ jobs:
           GCP_DEVNET_SERVICE_ACCOUNT: ${{ secrets.GCP_DEVNET_SERVICE_ACCOUNT }}
           GCP_DEVNET_SERVICE_ACCOUNT_KEY: ${{ secrets.GCP_DEVNET_SERVICE_ACCOUNT_KEY }}
 
-          # TODO: if not main -> dry run else deploy
```

Comment:
not sure, it is still actual. it should check on branch before main deploy. maintaining things other way is pain.